build.jam

```diff
@@ -0,0 +1,22 @@
+# Copyright René Ferdinand Rivera Morell 2023
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+project /boost/variant2
+    : common-requirements
+        <library>/boost/assert//boost_assert
+        <library>/boost/config//boost_config
+        <library>/boost/mp11//boost_mp11
+        <include>include
+    ;
+
+explicit
+    [ alias boost_variant2 ]
+    [ alias all : boost_variant2 test ]
```

Comment:
Do we need the `all` target? Why?

Reply:
As a general rule, `test` already depends on the library, so there's no need for `all`. If it doesn't, it's not much of a test.